x-pack/plugins/cases/server/client/cases/utils.ts

```diff
@@ -76,6 +77,13 @@ const getCommentContent = (comment: CommentResponse): string => {
   } else if (comment.type === CommentType.alert || comment.type === CommentType.generatedAlert) {
     const ids = getAlertIds(comment);
     return `Alert with ids ${ids.join(', ')} added to case`;
+  } else if (comment.type === CommentType.actions) {
+    const firstHostname = comment.actions.targets[0]?.hostname;
```

Comment:
Are we ok with this being the string `undefined` or should we default it to like `unknown` or something if `targets[0]` is undefined?